packages/amplication-server/src/core/prismaSchemaParser/prismaSchemaParser.service.ts

```diff
@@ -1352,7 +1350,7 @@ export class PrismaSchemaParserService {
 
     const enums = schema.list.filter((item) => item.type === ENUM_TYPE_NAME);
     const enumOfTheField = enums.find(
-      (item: Enum) => item.name === field.name
+      (item: Enum) => item.name === field.fieldType
```

Comment:
I don't understand this change. Did you also mean to change `item.name` to `item.type`? Or I am just not understanding what the data looks like here.